CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 0.6.1
+
+* #112 - bugfix to work as intended.
```

Comment:
I suggest describing what was fixed. This comment isn't very helpful without digging into the related code changes.